.cirrus.yml

```diff
@@ -4,7 +4,7 @@ task:
   name: FreeBSD
   env:
     matrix:
-      - JULIA_VERSION: 1.4
+      - JULIA_VERSION: 1.6
```

Comment:
You also need to update https://github.com/JuliaSmoothOptimizers/LimitedLDLFactorizations.jl/blob/a824f6b2b0d5ef2daba127cece6e6a7a7c0f49a4/Project.toml#L11